src/WordPress.php

```diff
@@ -476,35 +476,79 @@ private function generate_default_conf( $site_type, $cache_type, $server_name )
 	}
 
 
+	/**
+	 * Verify if the passed database credentials are working or not.
+	 *
+	 * @throws \Exception
+	 */
 	private function maybe_verify_remote_db_connection() {
 
-		if ( in_array( $this->site_data['db_host'], [ 'db', GLOBAL_DB ] ) ) {
+		if ( in_array( $this->site_data['db_host'], [ 'db', GLOBAL_DB ], true ) ) {
 			return;
 		}
+		$db_host        = $this->site_data['db_host'];
+		$img_versions   = \EE\Utils\get_image_versions();
+		$container_name = \EE\Utils\random_password();
+		$network        = ( GLOBAL_DB === $this->site_data['db_host'] ) ? "--network='" . GLOBAL_FRONTEND_NETWORK . "'" : '';
+
+		if ( ! \EE::exec( sprintf( 'docker run --name %s %s -e MYSQL_ROOT_PASSWORD=%s -d --restart always easyengine/mariadb:%s', $container_name, $network, \EE\Utils\random_password(), $img_versions['easyengine/mariadb'] ) ) ) {
```

Comment:
Please move this sprintf to another variable and use it here for better readability. Like you've done below in `$check_db_exists` variable.